carburetor-whiskers.json

```diff
@@ -0,0 +1,86 @@
+{
```

Comment:
Sorry for the nit, but could probably have this in `examples/whiskers.json`

Otherwise lgtm!

Reply:
Possibly, I'd argue you might want to rename the `examples` directory in that case though. The stuff in there right now is less "examples" and more "here, it's done for you already". Not that it matters too much as long as the documentation links to it properly.

Reply:
I agree, it was previously `src`, I might move it back to that